src/Conversion/ONNXToKrnl/Math/RandomNormal.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===----------------- Matmul.cpp - Lowering Matmul Op --------------------===//
+//
+// Copyright 2019 The IBM Research Authors.
```

Comment:
Do we need the copyright?  If we do fix 2019 -> 2021  (time flies when working on a fun project :-))

Reply:
Name is also RandomNormal.cpp